source/_patterns/02-organisms/teasers/teasers.twig

```diff
@@ -0,0 +1,21 @@
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['teasers']) }) -%}
+
+<div {%- include('atoms-attributes') -%}>
```

Comment:
Should be an `<ol>`. We'll need a heading too.

Reply:
> Should be an `<ol>`. We'll need a heading too.

I'm removing the teasers list from the feature branch as we have a followup ticket for that: https://github.com/libero/libero/issues/114